industry_restaurant/data/ir_default_data.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
data is another level, useful when dealing with updatable and non-updatable records. Here, there is only one type, so only odoo is fine
```suggestion
<odoo noupdate="1">
```